Makefile

```diff
@@ -1,8 +1,17 @@
-build:
-	./node_modules/pegjs/bin/pegjs -o pegjs-generated.js rules.pegjs
+build: rules-gen.pegjs
+	./node_modules/pegjs/bin/pegjs rules-gen.pegjs pegjs-generated.js
```

Comment:
I didn't get it working with out `-o`. Which node version are you using? I am using v7.